Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -35,7 +35,40 @@ final class MKMapViewTests: XCTestCase {
         let annotationViewWithAnnotation = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self, for: annotation)
         XCTAssertNotNil(annotationViewWithAnnotation)
     }
+    
+    func testWithEmptyItemArray() {
+        let mapView = MKMapView()
+        let meter = 500.0
+        let emptyItemArray = [CLLocationCoordinate2D]()
+        let edgePadding = UIEdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+        mapView.zoom(to: emptyItemArray, meter: meter, edgePadding: edgePadding, animated: true)
 
+        XCTAssert(true)
```

Comment:
This still doesn't feel right, consider this:

```swift
let mapView = MKMapView()
let meter = 500.0
let edgePadding = UIEdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
let previous = mapView.visibleMapRect
mapView.zoom(to: [], meter: meter, edgePadding: edgePadding, animated: true)
// So here we are testing that invoking this function with an empty array, doesn't affect the mapview state :))
XCTAssert(previous, mapView.visibleMapRect)
```
That way we are basically testing that calling that extension with empty array doesn't do nothing to the state of the map view.

Reply:
cc @mustafagunes

Reply:
Thank you for the solution @LucianoPAlmeida 🙏 . I'il make a commitment soon. I'm new to writing tests. Sorry 😞

Reply:
No problem! Happy to help

Reply:
@LucianoPAlmeida What I don't understand is if the method doesn't crash, shouldn't the test return `XCTAssert (true)` ?

Reply:
@LucianoPAlmeida Because above it will always turn false.

Reply:
Ohh, my bad ... I meant `XCTAssertEquals(previous, mapView.visibleMapRect)` :))